src/app/shared/wysiwyg/wysiwyg.component.ts

```diff
@@ -70,8 +70,7 @@ export class WysiwygComponent implements OnInit, OnChanges, OnDestroy {
 
   ngOnChanges(changes: SimpleChanges) {
     if (this.editor) {
-      let imagesChanged = changes['images'] && changes['images'].currentValue.length !== changes['images'].previousValue.length;
-      if (imagesChanged) {
+      if (changes['images']) {
```

Comment:
This scares me just a little. I'll do some testing on it. When I built this, images seemed to always be in the changes even when there were no changes and it fires constantly, which means editor menu is being rebuilt with lots of DOM changes.